### PR TITLE
RUBY-2984 Retry DNS seed list tests

### PR DESCRIPTION
**Pull request by p-mongo**

### CODE REVIEW

File: spec/spec_tests/seed_list_discovery_spec.rb [hunk 1]
```diff
@@ -78,6 +82,8 @@
           end
 
           if test.expected_options
+            # In Evergreen sometimes this test fails intermittently.
+            retry_test
```
Comment (Neilshweky) [inline spec/spec_tests/seed_list_discovery_spec.rb:86]: Can I remove this one since it's on the context on line 57?

**Comment by Neilshweky** [comment]
Oleg what's the status of this PR?

File: spec/spec_tests/seed_list_discovery_spec.rb (hunk 1 above)
Comment (p-mongo) [reply spec/spec_tests/seed_list_discovery_spec.rb:86]: Yes I think so, I've done this.

**Comment by p-mongo** [comment]
I removed libmongocrypt changes from this PR, now it just covers https://jira.mongodb.org/browse/RUBY-2984.

**Pull request merged**